wdio.conf.js

```diff
@@ -0,0 +1,395 @@
+                            'const { Linter } = require("../../../lib/linter");',
+                            'const { Linter } = require("../../../build/eslint");\n' +
+                            'process.cwd = () => "/";'
+                        );
+                    }
+                }
+            ]
+        }
+    }],
+
+    /*
+     *
+     * ==================
+     * Specify Test Files
+     * ==================
+     * Define which test specs should run. The pattern is relative to the directory
+     * of the configuration file being run.
+     *
+     * The specs are defined as an array of spec files (optionally using wildcards
+     * that will be expanded). The test for each spec file will be run in a separate
+     * worker process. In order to have a group of spec files run in the same worker
+     * process simply enclose them in an array within the specs array.
+     *
+     * If you are calling `wdio` from an NPM script (see https://docs.npmjs.com/cli/run-script),
+     * then the current working directory is where your `package.json` resides, so `wdio`
+     * will be called from there.
+     *
+     */
+    specs: [
+        "tests/lib/linter/linter.js"
+    ],
+
+    // Patterns to exclude.
+    exclude: [],
+
+    /*
+     *
+     * ============
+     * Capabilities
+     * ============
+     * Define your capabilities here. WebdriverIO can run multiple capabilities at the same
+     * time. Depending on the number of capabilities, WebdriverIO launches several test
+     * sessions. Within your capabilities you can overwrite the spec and exclude options in
+     * order to group specific specs to a specific capability.
+     *
+     * First, you can define how many instances should be started at the same time. Let"s
+     * say you have 3 different capabilities (Chrome, Firefox, and Safari) and you have
+     * set maxInstances to 1; wdio will spawn 3 processes. Therefore, if you have 10 spec
+     * files and you set maxInstances to 10, all spec files will get tested at the same time
+     * and 30 processes will get spawned. The property handles how many capabilities
+     * from the same test should run tests.
+     *
+     */
+    maxInstances: 10,
+
+    /*
+     *
+     * If you have trouble getting all important capabilities together, check out the
+     * Sauce Labs platform configurator - a great tool to configure your capabilities:
+     * https://saucelabs.com/platform/platform-configurator
+     *
+     */
+    capabilities: [{
+        browserName: "chrome",
+        "goog:chromeOptions": {
+            args: process.env.CI ? ["headless", "disable-gpu"] : []
+        }
+    }],
+
+    /*
+     *
+     * ===================
+     * Test Configurations
+     * ===================
+     * Define all options that are relevant for the WebdriverIO instance here
+     *
+     * Level of logging verbosity: trace | debug | info | warn | error | silent
+     */
+    logLevel: "info",
+    outputDir: "./logs",
```

Comment:
nits: I prefer something like `"./wdio-logs"`, to avoid conflicting with other tools. :)